public/tracker/vaccine_table.csv

```diff
@@ -1,6 +1,6 @@
 No Vaccine,1.00,1.00,1.00
-"Unknown vaccine, unknown date",0.40,0.40,0.40
-Pfizer,1.00,0.56,0.10
-Moderna,1.00,0.56,0.10
-AstraZenica,1.00,0.56,0.40
-"Johnson & Johnson",1.00,0.33,0.33
+"Unknown vaccine, unknown date",0.47,0.47,0.47
+Pfizer,1.00,0.76,0.16
+Moderna,1.00,0.76,0.16
+AstraZenica,1.00,0.76,0.47

```

Comment:
What is this .csv used for? Will changing it to AstraZen**E**ca break anything? (feel free to defer.)

Reply:
It's used for the risk tracker. It imports this from the site. And yes, changing the name will break it. Which sucks. So... I guess we should leave it?